src/main/java/com/epam/preprod/sirenko/CreatedByInputProductImpl.java

```diff
@@ -0,0 +1,126 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+import com.epam.preprod.sirenko.util.ConsoleReader;
+import com.epam.preprod.sirenko.util.PrintToConsole;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+public class CreatedByInputProductImpl implements Strategy {
```

Comment:
Maybe we can use better naming for example ManualProductCreationStrategy

Reply:
done